src/engine/BMInterfaceGame.php

```diff
@@ -49,23 +48,43 @@ public function create_game(
         }
 
         try {
-            $gameId = $this->insert_new_game($playerIdArray, $maxWins, $description, $previousGameId);
+            if (!isset($currentPlayerId)) {
+                if ($this->isTest) {
```

Comment:
Yeah, that latter thing sounds like the wrapper idea, and probably makes your life easier here, because you can literally find/replace the calls to create_game with test_create_game_wrapper or whatever, and have that function set `$currentPlayerId` if it isn't set.  That may or may not be what you want for all invocations, so putting the change in the test code gives you the options.

For instance, suppose you want to make a test that demonstrates that `create_game()` fails in a clean way if `$currentPlayerId` isn't set?  Right now, you can't write that test.  But if you had a wrapper, you could simply not use the wrapper for that particular test.